src/librustc_metadata/schema.rs

```diff
@@ -375,14 +376,21 @@ impl<'a, 'gcx> HashStable<StableHashingContext<'a>> for EntryKind<'gcx> {
             EntryKind::AssociatedType(associated_container) => {
                 associated_container.hash_stable(hcx, hasher);
             }
-            EntryKind::AssociatedConst(associated_container, qualif) => {
+            EntryKind::AssociatedConst(associated_container, qualif, _) => {
                 associated_container.hash_stable(hcx, hasher);
                 qualif.hash_stable(hcx, hasher);
             }
         }
     }
 }
 
+#[derive(RustcEncodable, RustcDecodable)]
+pub struct ConstData {
+    pub rendered: String,
```

Comment:
Is this expected to gain more fields? If no, how about making t a newtype called `RenderedConst`?

Also 🚨documentation police🚨 please add some docs

Reply:
I'm not aware of any additional data we need at this time. I'd be happy to change it and I'll add some docs too 👍